docs/sources/lids/template.md

```diff
@@ -0,0 +1,51 @@
+# Title
+
+> _NOTE: the file should be named `_<your-title>.md` and be placed in the `docs/sources/lids` directory. 
+Once accepted, it will be assigned a LID number and the file will be renamed by the sponsor.<br>
+> **Please remove this section before submitting, and retain the structure of this document.**_
+
+**Author:** Name (email)
+
+**Date:** MM/YYYY
+
+**Sponsor(s):** @username of maintainer(s) willing to shepherd this LID
+
+**Type:** Feature|Process
+
+**Status:** Draft|In Discussion|Accepted|Rejected|Completed
+
+**Related issues/PRs:**
+
+**Thread from [`#loki-dev` channel on Slack](https://slack.grafana.com):**
+
+---
+
+## Background
+
+_Describe the key facts and link to any documents, architecture diagrams or code. Keep this short, just a paragraph or two._
+
+## Problem Statement
+
+_Describe why the above is a problem we have now or will have in the future. Just a paragraph or two._
+
+## Goals
+
+_Describe the goals that we wish to achieve when considering the options available. Be clear about what is important and what could be left to the future. Non-goals are also useful to list here to clarify intent._
+
+## Proposals
+
+_List at least two possible ways of achieving the goals, these are typically different technical approaches that have differing trade-offs so summarise the trade-offs. “Do nothing” is typically worth considering too._
+
+### Proposal 0: Do nothing
+
+_Describe what happens if we do nothing, how bad does the problem become (over what time frame)?_
+
+### Proposal 1: Title
+
+_Describe the first proposal, what are the benefits and trade-offs that this approach has?_
+
+### Proposal n: Title
+
+_Describe the nth proposal(s), what are the benefits and trade-offs that these approaches have?_
+
+## Other Notes
```

Comment:
I also suggest a two additional sections here:
- `Implementation history`: It might take a contributor multiple attempts to get an implementation right across Loki releases even if the LID is the right thing to do. Loki development is a constant moving target and one can easily introduce something that works well in microservices but not well in SSD or else.
- `Drawbacks`: In contrast to the Do-Nothing Section this one should reflect the hard bites we are willing to take by accepting a LID. For example the introduction of SSD introduced a third mode to continuously maintain.

Reply:
Ditto as above